Doc/library/py_compile.rst

```diff
@@ -82,6 +89,10 @@ byte-code cache files in the directory containing the source code.
       overrides the value of the *invalidation_mode* argument, and determines
       its default value instead.
 
+   .. versionchanged:: 3.8
+      The *quiet* parameter was added to support silent mode. It returns full
```

Comment:
We've already described the new parameter in the main documentation. We can just say "\*quiet\* parameter was added." here.